### PR TITLE
Fix typo equipment

### PR DESCRIPTION
**Pull request by koshikawa**
Fix typo equipment

### CODE REVIEW

File: inc/config.class.php [hunk 1]
```diff
@@ -466,7 +466,7 @@ function changestatus() {
       echo "</tr>";
 
       echo "<tr class='tab_bg_1'>";
-      echo "<td>".__('Network equpment')."</td>";
+      echo "<td>".__('Network equipment')."</td>";
```
Comment (trasher) [inline inc/config.class.php:469]: This string does not seems to exists in core, so:
```suggestion
      echo "<td>".__('Network equipment', 'fusioninventory')."</td>";
```

**Review by trasher — CHANGES_REQUESTED** [review]
See comment

**Comment by koshikawa** [comment]
> See comment

OK

**Review by trasher — APPROVED** [review]
Thank you

**Review by orthagh — APPROVED** [review]

**Pull request merged**